### DISCUSSION TITLE
pgsodium / TCE (Transparent Column Encryption) not recommended and deprecated.

### DISCUSSION THREAD

**Discussion by pateljoel** [supabase/supabase#27109] [Questions]
I have noticed that Supabase highly does not recommend pgsodium for new projects or transparent column encryption anymore and are removing pgsodium.

https://supabase.com/docs/guides/database/extensions/pgsodium

I was looking for a way to encrypt sensitive data (think, passport information, addresses, drivers license info etc) for an MVP in supabase and would have thought that using pgsodium would be a good fit for this but pgsodium will be removed soon.

What should I use instead to encrypt columns in Supabase, or should I trust Supabase to encrypt all of the data I store in them at rest?

Please advise, thank you.

**Comment by tengweiherr** [comment]
> I have noticed that Supabase highly does not recommend pgsodium for new projects or transparent column encryption anymore and are removing pgsodium.
> 
> 
> 
> https://supabase.com/docs/guides/database/extensions/pgsodium
> 
> 
> 
> I was looking for a way to encrypt sensitive data (think, passport information, addresses, drivers license info etc) for an MVP in supabase and would have thought that using pgsodium would be a good fit for this but pgsodium will be removed soon.
> 
> 
> 
> What should I use instead to encrypt columns in Supabase, or should I trust Supabase to encrypt all of the data I store in them at rest?
> 
> 
> 
> Please advise, thank you.

What did you go with eventually? I am considering AWS KMS but I wonder if it could be done within Supabase
